src/components/Sidebar.vue

```diff
@@ -20,20 +31,20 @@ export default{
             <span v-if="collapsed">
             <div class="wrapper">TC</div>
             </span>
-            <span v-else><div class="wrapper">Travel Companion</div></span> 
+            <span v-else><div class="wrapper">Travel Companion</div></span>
         </h2>
         <h5>
         <!-- Sidebar Links -->
-        <SidebarLink to="/" icon="fas fa-home">Home</SidebarLink>
-        <SidebarLink to="/signIn" icon="fas fa-right-to-bracket">Sign In</SidebarLink>
-        <SidebarLink to="/tripManager" icon="fas fa-plane">Trip Manager</SidebarLink>
+        <SidebarLink v-if="username" to="/" icon="fas fa-home">Home</SidebarLink>
+        <SidebarLink v-if="!username" to="/signIn" icon="fas fa-right-to-bracket">Sign In</SidebarLink>
+        <SidebarLink v-if="username" to="/tripManager" icon="fas fa-plane">Trip Manager</SidebarLink>
+        <SidebarLink v-if="username" icon="fas fa-sign-out" @click="signUserOut">Sign Out</SidebarLink>
```

Comment:
Does this remove the sign in option when user is signed in and replace it with the sign out button?

Reply:
Yrah, so when the user is logged out, the only option they see is sign in. When they log in, everything else appears and the sign in disappears and is replaced with the sign out.

Reply:
Awesome! This was something I was concerned about when making the bar.